README.md

```diff
@@ -22,9 +22,8 @@ repositories {
     mavenCentral()
 }
 
-// The Gateway Connection Config is needed by the gateway-export plugin in order to export from a gateway
-GatewayConnection {
-    url = 'https://<gateway-host>:8443/restman'
+// The Gateway Export Config is needed by the gateway-export plugin in order to export from a gateway
+GatewayExportConfig {
```

Comment:
I think you should mention `GatewayConnection` section as well with the URL configuration.
just say that it's required only if it's not localhost:8443.